### ISSUE TITLE
Doesn't detect anything, can't quit.

### ISSUE THREAD

**Issue by gdonald** [1kc/razer-macos#636]
Doesn't detect my Razer DeathAdder v2 Pro on my Macbook Pro M1.

When I gave up waiting I was unable to quit the app.  I couldn't select any task bar items.  The app doesn't appear on my dock either.  I had to kill it using Activity Monitor.

Horrible user experience.

**Comment by unclekhalifa** [comment]
I had the same issue the first time I started it up and tried to get it to work with a Huntsman V2 TKL. Restarted it and realised there's a new menu item for the app.

<img width="540" alt="Screenshot 2022-05-13 at 20 07 53" src="https://user-images.githubusercontent.com/9752838/168373808-3a16233a-f5d2-4bb2-a53d-9e01ade5fac0.png">

**Comment by eDave56** [comment]
Yeah, it's the menu bar access through the three-square icon. Give it another go - you might have it working fully and just not realize it. 

With 10.4.5, which runs on my Mojave install, it's stable and seems fully functional, it just doesn't have the device I want in the library (Blackwidow v3 Mini keyboard). Sadly, my programming kung fu is woefully inadequate to the task of compiling the latest source for Mojave, particularly after first trying it with one of the most recent versions of XCode.

**Comment by gdonald** [comment]
Closing due to lack of interest.
